### DISCUSSION TITLE
Sentry React navigation integration

### DISCUSSION THREAD

**Discussion by zackify** [expo/router#665] [General]
Expose the navigation container so we can use the sentry react navigation integration:

https://docs.sentry.io/platforms/react-native/performance/instrumentation/automatic-instrumentation/#react-navigation

**Comment by wcastand** [comment]
can we get an update on how to do this from expo team?

**Comment by EvanBacon** [comment]
You can access the navigation ref with `useRootNavigation()`. It doesn't look like the Sentry team has an official integration for Expo Router though so YMMV.
